cpp/include/cudf/types.h

```diff
@@ -72,27 +72,50 @@ typedef enum {
     GDF_HASH_IDENTITY,  /**< Identity hash function that simply returns the key to be hashed */
 } gdf_hash_func;
 
+
+/**
+ * @brief Defines the unit of time that an algoithm or structure is storing.
+ *
+ * There are time types in cudf. Those time types can have different resolutions so this
+ * enum allows us to define different units of time. The types included are nanosecond,
+ * microsecond, millisecond, and second.
+ */
```

Comment:
```suggestion
 --------------------------------------------------------------------------- */
```

Reply:
And etc. on all comment blocks, to match existing comment blocks.

Reply:
Now that I look I see we're not consistent with these horizontal lines anymore... So take your pick.